src/app/pages/sign-stacks-message-request/use-sign-stacks-message-request.ts

```diff
@@ -70,7 +70,7 @@ export function useSignStacksMessageRequest() {
   if (!tabId) throw new Error('Requests can only be made with corresponding tab');
   if (!requestToken) throw new Error('Missing request token');
 
-  const signMessage = whenWallet({
+  const onSignMessage = whenWallet({
```

Comment:
These methods I'd have left as is, as they're not the event called to trigger signing the message, but the signing logic itself

Reply:
Ah right!